zimports/ventored.py

```diff
@@ -0,0 +1,183 @@
+"""Vendored code from other packages"""
+import collections
+import fnmatch as _fnmatch
+import os
+import re
+
+# ## flake8 code ##
```

Comment:
I'm not sure if this file should include the flake8 license at the top? @sigmavirus24 @zzzeek

Reply:
I think Flake8's MIT so I don't think it's entirely necessary but probably doesn't hurt? I don't remember any of this.